src/components/events/CheckInModal/style.module.scss

```diff
@@ -0,0 +1,83 @@
+@use 'src/styles/vars.scss' as vars;
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  max-width: 50rem;
+  padding: 2rem;
+
+  .graphic {
+    aspect-ratio: 560 / 283;
+    position: relative;
+    width: 100%;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: column;
+    gap: 0.25rem;
+
+    .headerText {
+      text-align: center;
+
+      @media screen and (width <= vars.$breakpoint-md) {
+        font-size: 2rem !important;
+        line-height: 2.5rem !important;
+      }
+    }
+
+    .subheader {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+
+      .subheaderText {
+        text-align: center;
+
+        @media screen and (width <= vars.$breakpoint-md) {
+          font-size: 1rem !important;
+          line-height: 1.5rem !important;
+        }
+      }
+    }
+
+    .close {
+      align-items: center;
+      background: none;
+      display: flex;
+      height: 3.75rem;
+      margin-left: 1rem;
+      padding: 0;
+    }
+  }
+
+  .image {
+    aspect-ratio: 1920 / 1080;
+    border-radius: 10px;
+    flex-shrink: 0;
+    overflow: hidden;
+    position: relative;
+    width: 100%;
+  }
+
+  .done {
```

Comment:
The done button looks nice but I feel like it doesn't fit with the existing design system's buttons. I think at the very least it should have the same font weight so it doesn't look like the body text